asset-transfer-basic/rest-api-typescript/scripts/generateEnv.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+: "${TEST_NETWORK_HOME:=../..}"
```

Comment:
Just my opinion of course, but I really wish we had a better way of doing this.

Reply:
Same here! Suggestions welcome :)

Reply:
some sort of card?